### DISCUSSION TITLE
第2节课作业(6班)

### DISCUSSION THREAD

**Discussion by zhangzefeng92** [InternLM/Tutorial#159] [作业]
微信名称：捂上眼睛看世界

**基础作业：**

1. 使用 InternLM-Chat-7B 模型生成 300 字的小故事（需截图）。
![图片](https://github.com/InternLM/tutorial/assets/58167242/1b9c861a-47d7-4f0f-8d77-78ac4b4e6e78)

2. 熟悉 hugging face 下载功能，使用 huggingface_hub python 包，下载 InternLM-20B 的 config.json 文件到本地（需截图下载过程）。
![图片](https://github.com/InternLM/tutorial/assets/58167242/5c8725db-d900-4481-a83c-f1272d5c044d)

注：huggingface国内无法访问解决方案https://hf-mirror.com/

**进阶作业**

1. 完成浦语·灵笔的图文理解及创作部署（需截图）
![图片](https://github.com/InternLM/tutorial/assets/58167242/8b31ba6d-e1d8-469f-b584-0e595df18cfb)

注：Could not create share link. Missing file: /root/.local/lib/python3.10/site-packages/gradio/frpc_linux_amd64_v0.2. 解决方案

- 1. Download this file: https://cdn-media.huggingface.co/frpc-gradio-0.2/frpc_linux_amd64
- 2. Rename the downloaded file to: frpc_linux_amd64_v0.2
- 3. Move the file to this location: /root/.local/lib/python3.10/site-packages/gradio
- 4. chmod 777 /root/.local/lib/python3.10/site-packages/gradio/frpc_linux_amd64_v0.2

2. 完成 Lagent 工具调用 Demo 创作部署（需截图）
![图片](https://github.com/InternLM/tutorial/assets/58167242/14b1de7a-ad18-431b-8da8-4a530bf4fd91)

**Comment by JackieGLQ** [comment]
微信名：老高
基础作业：https://z01krqqa831.feishu.cn/docx/EMs2dSD0YoQpSExNKDHc3dY0n9g#FmiEdEVmHoRCREx3VLAcA6XjnEb
进阶作业：https://z01krqqa831.feishu.cn/docx/EMs2dSD0YoQpSExNKDHc3dY0n9g#HPFkdqnsHo15LuxwA0wcd5HKn3e
                  https://z01krqqa831.feishu.cn/docx/EMs2dSD0YoQpSExNKDHc3dY0n9g#StrKd5A3doCXmixuYotcX5ronTf

**Comment by yokebtc** [comment]
yoke - [https://app.yinxiang.com/fx/fa88c3db-ec90-4389-a7db-a78f26038ac3](https://app.yinxiang.com/fx/fa88c3db-ec90-4389-a7db-a78f26038ac3)

**Comment by RalphGuo** [comment]
guos-https://gitee.com/RalphGs/InternLM-assignment/tree/master/%E8%AF%BE%E7%A8%8B2
